autopush/tests/test_websocket.py

```diff
@@ -51,6 +51,20 @@
     "content-encoding": "aesgcm",
     "TTL": "60",
 }
+dummy_chid = str(uuid.uuid4())
+dummy_uaid = str(uuid.uuid4())
```

Comment:
`dummy_uaid` is only used once. Does it need to be a string or can it stay a UUID?